exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/propagator.py

```diff
@@ -42,6 +42,9 @@ def extract(
         context: typing.Optional[Context] = None,
         getter: Getter = default_getter,
     ) -> Context:
+        if context is None:
+            context = Context()
```

Comment:
In this PR description you mention this, @mariojonke:

_Propagators should return the passed in context (or the current one if none was given) if nothing could be extracted._

Should we use the current context instead of instantiating a new one?

Reply:
We decided in the SIG meeting to use the root context as default because the current context would continue a possible active trace instead of starting a new one if extracting from the carrier fails  (see also https://github.com/open-telemetry/opentelemetry-python/issues/1765#issuecomment-827464045)

See also PR in the core repo: https://github.com/open-telemetry/opentelemetry-python/pull/1811

Reply:
Ok, good :+1: